shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -373,15 +375,17 @@ public void process(ReplyProcessor cp) {
         public void writeFields(DataOutput out) throws IOException {
             out.writeLong(writerId.getMostSignificantBits());
             out.writeLong(writerId.getLeastSignificantBits());
+            out.writeLong(requestId);
             out.writeLong(eventNumber);
```

Comment:
This needs to go last

Reply:
Done. Moved it to last to make backward compatible.